src/components/input/input.spec.js

```diff
@@ -346,6 +346,29 @@ describe('md-input-container directive', function() {
     expect(label.textContent).toEqual('some placeholder');
   });
 
+  it('should not create a floating label from a placeholder if md-no-float exists and is empty, or evaluates to true', function () {
```

Comment:
Could you add that test real fast, or perhaps update the existing test's description to just say `md-no-float evaluates to true`? :smile: